src/main/resources/com/sumologic/sumobot/http_frontend/script.js

```diff
@@ -18,23 +18,39 @@
  */
 window.onload = function() {
     var websocketProtocol = window.location.protocol === "https:" ? "wss://" : "ws://";
-
     var endpoint = websocketProtocol + window.location.host + window.location.pathname + "websocket";
-
     var socket = new WebSocket(endpoint);
-    var messageBox = document.getElementById("messages");
 
+    var messageBox = document.getElementById("messages");
+    var messageInput = document.getElementById("message-input");
     var submitButton = document.getElementById("submit");
+
+    function appendMessage(msg, styles) {
```

Comment:
`styles` is really a list of classes here, I'd rename the parameter to classes. You can use the `classList` API to operate on classes like that, works even in IE 10+ (except in SVGs).

I'd also make it an array of classes, not a string as it appears in HTML.